SConstruct

```diff
@@ -183,6 +183,7 @@ opts.Add(BoolVariable("separate_debug_symbols", "Extract debugging symbols to a
 opts.Add(EnumVariable("lto", "Link-time optimization (production builds)", "none", ("none", "auto", "thin", "full")))
 opts.Add(BoolVariable("production", "Set defaults to build Godot for use in production", False))
 opts.Add(BoolVariable("generate_apk", "Generate an APK/AAB after building Android library by calling Gradle", False))
+opts.Add(BoolVariable("generate_bundle", "Generate an APP bundle after building iOS/macOS binaries", False))
```

Comment:
These could maybe be added in each platform's `detect.py` (`get_opts`).

This also applies to `generate_apk`, I missed it when reviewing that PR. CC @Calinou

Reply:
WDYT @bruvzg?

Reply:
Rebased and move both options to the platforms.